packages/back-end/src/models/FeatureRevisionModel.ts

```diff
@@ -118,6 +138,23 @@ export async function getRevision(
   return doc ? toInterface(doc) : null;
 }
 
+export async function getRevisionsByStatus(
+  context: ReqContext,
+  statuses: string[]
+) {
+  const revisions = await FeatureRevisionModel.find({
+    id: context.org.id,
+    status: { $in: statuses },
```

Comment:
Every single query we run must always filter by `organization`.  This is critical for security since it prevents us from leaking information between organizations.